src/components/icons/TooltipIcon.tsx

```diff
@@ -5,12 +5,15 @@ export const TooltipIcon = createIcon({
   displayName: 'Tooltip',
   viewBox: '0 0 24 24',
   path: (
-    <path
-      fillRule="evenodd"
-      clipRule="evenodd"
-      d="M6.4443 20.3147C8.08879 21.4135 10.0222 22 12 22C14.6522 22 17.1957 20.9464 19.0711 19.0711C20.9464 17.1957 22 14.6522 22 12C22 10.0222 21.4135 8.08879 20.3147 6.4443C19.2159 4.79981 17.6541 3.51809 15.8268 2.76121C13.9996 2.00433 11.9889 1.8063 10.0491 2.19215C8.10929 2.578 6.32746 3.53041 4.92894 4.92894C3.53041 6.32746 2.578 8.10929 2.19215 10.0491C1.8063 11.9889 2.00433 13.9996 2.76121 15.8268C3.51809 17.6541 4.79981 19.2159 6.4443 20.3147ZM7.55544 5.34825C8.87104 4.4692 10.4178 4 12 4C14.1217 4 16.1566 4.84286 17.6569 6.34315C19.1572 7.84344 20 9.87827 20 12C20 13.5823 19.5308 15.129 18.6518 16.4446C17.7727 17.7602 16.5233 18.7855 15.0615 19.391C13.5997 19.9965 11.9911 20.155 10.4393 19.8463C8.88743 19.5376 7.46197 18.7757 6.34315 17.6569C5.22433 16.538 4.4624 15.1126 4.15372 13.5607C3.84504 12.0089 4.00347 10.4003 4.60897 8.93854C5.21447 7.47673 6.23985 6.2273 7.55544 5.34825ZM13 11L11 12V18H13V11ZM13 6L11 7V9H13V6Z"
-      fill="currentColor"
-    />
+    <>
+      <title>Tool tip icon</title>
```

Comment:
is `tooltip` one word?

Reply:
I don't know if it's one word. I could check that.

Reply:
I corrected it know, "Tooltip" is one word.